client/my-sites/customer-home/cards/actions/quick-links/index.jsx

```diff
@@ -4,13 +4,10 @@
 import React from 'react';
```

Comment:
Note that these changes are actually caused by renaming `quick-links-compact` to `quick-links`.